ratis-test/src/test/java/org/apache/ratis/datastream/TestDataStreamBase.java

```diff
@@ -206,15 +192,15 @@ void runTestDataStream(int bufferSize, int bufferNum) {
       final DataStreamReply reply = impl.getHeaderFuture().join();
       Assert.assertTrue(reply.isSuccess());
       Assert.assertEquals(0, reply.getBytesWritten());
-      Assert.assertEquals(reply.getType(), Type.STREAM_HEADER);
+      Assert.assertEquals(reply.getType(), DataStreamPacketHeaderProto.Type.STREAM_HEADER);
```

Comment:
why need this change ?

Reply:
Ah my IDE somehow made this change. I can revert it.

Reply:
Reverted.